xml/dhcp_vendor_class.xml

```diff
@@ -0,0 +1,215 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<fingerprints matches="dhcp_vendor_class" protocol="dhcp" database_type="service">
+  <!--
+    Fingerprint definitions that are matched against the string values in the
+    dhcp message vi_vendor_class field
+    The vi_vendor_class field can be found in client discover (1), request (3)
+    and inform (8) messages.
+  -->
+
+  <fingerprint pattern="^Mfg=(?:Fuji)?(?i:Xerox);Typ=(?:MFP|printer);Mod=(?:Xerox )?(\S+) ([a-zA-Z0-9]+)">
+    <description>Xerox Multifunction Printer</description>
+    <example hw.family="VersaLink" hw.model="C405">Mfg=Xerox;Typ=MFP;Mod=VersaLink C405;Ser=</example>
+    <example hw.family="AltaLink" hw.model="C8035">Mfg=Xerox;Typ=MFP;Mod=Xerox AltaLink C8035 Multifunction Printer;Ser=</example>
+    <example hw.family="WorkCentre" hw.model="3345">Mfg=XEROX;Typ=MFP;Mod=WorkCentre 3345;Ser=3BT844742;Loc=</example>
+    <example hw.family="Phaser" hw.model="6500DN">Mfg=FujiXerox;Typ=printer;Mod=Phaser 6500DN;Ser=YXE135427</example>
```

Comment:
We will probably want to capture the serial numbers here but that can wait until this Issue is sorted out:
https://github.com/rapid7/recog/issues/364